README.md

```diff
@@ -46,6 +46,14 @@ Second, when you extend the `.estlintrc` file, you will have to suffic the exten
 
 ## Changelog
 
+- **3.0.0**
```

Comment:
If I add a new rule that creates `errors`, shouldn't I update the major version since it would be a breaking change to introduce errors that may not have been there previously?